packages/charts/react-charting/etc/react-charting.api.md

```diff
@@ -39,6 +39,11 @@ export type ChartDataMode = 'default' | 'fraction' | 'percentage';
 // @public (undocumented)
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33299&commitId=21f441ed4a94b152e4754723dd0f3653d7c3f8c8&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Beak 25.chromium.png|2186| Changed|
|Callout.Top right edge.chromium.png|1594| Changed|
|Callout.Top center.chromium.png|2178| Changed|
|Callout.Right center.chromium.png|2396| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Top auto edge.chromium.png|2307| Changed|
|Callout.Left bottom edge.chromium.png|3022| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>